infra/src/main/resources/i18n/translations.fi.json

```diff
@@ -377,7 +381,32 @@
             "LKR": "LKR (Läntinen keskiraide)",
             "ITHR": "ItHR (Itäinen huoltoraide (Helsinki-Ilmala)",
             "LANHR": "LänHR (Läntinen huoltoraide (Helsinki-Ilmala)"
-        }
+        },
+        "RaideType": {
+            "LP": "Liikennepaikka (LP)",
+            "LPO": "Liikennepaikan osa (LPO)",
+            "OLP": "Osiinjaettu liikennepaikka (OLP)",
+            "SEIS": "Seisake (SEIS)",
+            "LVH": "Linjavaihde (LVH)"
+        },
```

Comment:
Ajattelin että näissä voitaneen leipoa lyhenteet osaksi käännöksiä koska näitä on niin vähän. Toinen vaihtoehto olisi tehdä alemman kaltainen ratkaisu